lib/xboxrt/libc_extensions/wchar.c

```diff
@@ -323,3 +323,103 @@ wchar_t * wmemmove( wchar_t * dest, const wchar_t * src, size_t n )
     }
     return rv;
 }
+
+wchar_t * wmemset(wchar_t *wcs, wchar_t wc, size_t n)
+{
+    wchar_t * p = wcs;
+    while ( n-- )
+    {
+        *p++ = wc;
+    }
+    return wcs;
+}
+
+size_t wcrtomb (char *mbchar, wchar_t wchar, mbstate_t *mbstate)
+{
+    assert(0);
+    return 0;
+}
+
+size_t mbrtowc (wchar_t *wchar, const char *mbchar, size_t count, mbstate_t *mbstate)
+{
+    assert(0);
+    return 0;
+}
+
+double wcstod(const wchar_t * XBOXRT_RESTRICT nptr, wchar_t ** XBOXRT_RESTRICT endptr)
+{
+    assert(0);
+    return 0.0;
+}
+
+float wcstof(const wchar_t * XBOXRT_RESTRICT nptr, wchar_t ** XBOXRT_RESTRICT endptr)
+{
+    assert(0);
+    return 0.0;
+}
+
+long double wcstold(const wchar_t * XBOXRT_RESTRICT nptr, wchar_t ** XBOXRT_RESTRICT endptr)
+{
+    assert(0);
+    return 0.0;
+}
+
+long wcstol(const wchar_t * XBOXRT_RESTRICT nptr, wchar_t ** XBOXRT_RESTRICT endptr, int base)
+{
+    assert(0);
+    return 0.0;
+}
+
+long long wcstoll(const wchar_t * XBOXRT_RESTRICT nptr, wchar_t ** XBOXRT_RESTRICT endptr, int base)
+{
+    assert(0);
+    return 0.0;
+}
+
+unsigned long wcstoul(const wchar_t * XBOXRT_RESTRICT nptr, wchar_t ** XBOXRT_RESTRICT endptr, int base)
+{
+    assert(0);
+    return 0.0;
+}
+
+unsigned long long wcstoull(const wchar_t * XBOXRT_RESTRICT nptr, wchar_t ** XBOXRT_RESTRICT endptr, int base)
+{
+    assert(0);
+    return 0.0;
```

Comment:
Returns `unsigned long long` but is stubbed with a non-integer value. Looks like a copy-paste mistake?

Reply:
Fixed all occurrences of this. Interesting that it didn't produce a warning before.